internal/cluster/engine.go

```diff
@@ -604,6 +606,30 @@ func (x *Engine) GetActor(ctx context.Context, actorName string) (*internalpb.Ac
 	return actor, nil
 }
 
+// ActorExists checks whether an actor exists in the cluster
+func (x *Engine) ActorExists(ctx context.Context, actorName string) (bool, error) {
+	// return an error when the engine is not running
+	if !x.IsRunning() {
+		return false, ErrEngineNotRunning
+	}
+
+	ctx, cancelFn := context.WithTimeout(ctx, x.readTimeout)
+	defer cancelFn()
+
+	x.Lock()
+	defer x.Unlock()
+
+	// check whether the actor exists in the actors map
+	_, err := x.actorsMap.Get(ctx, actorName)
```

Comment:
```suggestion
	if _, err := x.actorsMap.Get(ctx, actorName); err != nil {
	   if errors.Is(err, olric.ErrKeyNotFound) {
	     return false, nil
	   }
	   return false, err
	}
	return true, nil
```

The expectation is to return true.